src/windows/gfx/d3d11-ctx.c

```diff
@@ -77,16 +271,23 @@ static bool d3d11_ctx_init(struct d3d11_ctx *ctx)
 	IDXGIFactory2 *factory2 = NULL;
 	IDXGISwapChain1 *swap_chain1 = NULL;
 
+	ctx->format = MTY_COLOR_FORMAT_BGRA;
+	ctx->colorspace = MTY_COLOR_SPACE_SRGB;
+
 	DXGI_SWAP_CHAIN_DESC1 sd = {0};
-	sd.Format = DXGI_FORMAT_B8G8R8A8_UNORM;
+	sd.Format = DXGI_FORMAT_R8G8B8A8_UNORM;
 	sd.BufferUsage = DXGI_USAGE_RENDER_TARGET_OUTPUT;
 	sd.SwapEffect = DXGI_SWAP_EFFECT_FLIP_DISCARD;
 	sd.SampleDesc.Count = 1;
 	sd.BufferCount = 2;
 	sd.Flags = D3D11_SWFLAGS;
-
+	
 	D3D_FEATURE_LEVEL levels[] = {D3D_FEATURE_LEVEL_11_1, D3D_FEATURE_LEVEL_11_0};
-	HRESULT e = D3D11CreateDevice(NULL, D3D_DRIVER_TYPE_HARDWARE, NULL, 0, levels,
+	UINT flags = 0;
+	#ifdef DEBUG
```

Comment:
Add a flag `D3D11_CTX_DEBUG` (you can check out the `D3D12_CTX_DEBUG` version for it`)